data/HexHive/projects.yaml

```diff
@@ -483,15 +476,11 @@ projects:
     incubator:
       type: retired
       work: 2021/Q1 worked on improving and integrating with microG and published app
-      products:
-        - type: App
-          title: LibreCHovid
-          url: https://f-droid.org/en/packages/ch.corona.tracing/
     c4dt_contact:
       name: Christian Grigis
       email: christian.grigis@epfl.ch
     date_added: 2020-10-05
```

Comment:
shouldn't he be removed as contact since he's no longer part of C4DT?

Reply:
Removed this and other old contacts.